ogr/services/gitlab/service.py

```diff
@@ -123,3 +123,32 @@ def project_create(
         return GitlabProject(
             repo=repo, namespace=namespace, service=self, gitlab_repo=new_project
         )
+
+    def list_projects(
+        self,
+        namespace: str = None,
+        user: str = None,
+        search_patter: str = None,
+        language: str = None,
+    ) -> List[GitProject]:
+        gl = self.gitlab_instance
+
+        if namespace:
+            group = gl.groups.get(namespace)
+            projects = group.projects.list()
+        if user:
+            u = gl.users.list(username=user)[0]
+            projects = u.projects.list()
```

Comment:
I know that I am sometimes really strict about the code style..;) Can you please use the full name for variables? (The code is more read than written.)

And as I wrote above for `self.github`, you don't need to save `self.gitlab_instance` to a variable.